pkg/plugins/diff/diff.proto

```diff
@@ -0,0 +1,39 @@
+/*
+Run from lakeFS root:
+    protoc --proto_path=pkg/plugins/diff --go_out=pkg/plugins/diff --go_opt=paths=source_relative \
+        --go-grpc_out=pkg/plugins/diff --go-grpc_opt=paths=source_relative diff.proto
+ */
+syntax = "proto3";
+package diff;
+option go_package = "github.com/treeverse/lakefs/plugins/diff";
+
+message DiffRequest {
+  TablePaths paths = 1;
+  GatewayConfig gateway_config = 2;
+}
+
+message TablePaths {
+  string left_table_path = 1;
+  string right_table_path = 2;
+  optional string base_table_path = 3;
+}
```

Comment:
'_table_' looks redundant.
consider a comment about the path format.

Reply:
done